tests/xharness/XmlResultParser.cs

```diff
@@ -0,0 +1,267 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+
+namespace xharness {
+	public static class XmlResultParser {
+
+		public enum XmlResultType {
+			TouchUnit,
+			NUnit,
+			xUnit,
+		}
```

Comment:
minor: inner (public) types requires more typing to use and are less discoverable

In this case it also duplicate (part of) the name

either
* move it outside the type (and keep the name) - my preference
or
* rename it to something smaller, e.g. `Jargon` because `Type` alone would be confusing

so you don't have to write `XmlResultParser.XmlResultType` but `XmlResultParser.Jargon`